hudi-client/src/main/java/org/apache/hudi/io/compact/HoodieMergeOnReadTableCompactor.java

```diff
@@ -18,6 +18,7 @@
 
 package org.apache.hudi.io.compact;
 
+import com.google.common.collect.Sets;
```

Comment:
Can we put this back the original place?

Reply:
done